apimodels/test_results.go

```diff
@@ -27,29 +27,40 @@ type CedarTestResult struct {
 // GetCedarTestResultsOptions represents the arguments passed into the
 // GetCedarTestResults function.
 type GetCedarTestResultsOptions struct {
-	BaseURL   string
-	TaskID    string
-	TestName  string
-	Execution int
+	BaseURL       string
```

Comment:
Can we put `json:"-"` tags on this so that it's obvious that it's not a REST API model?

Reply:
I am not super opinionated about this, but I don't think it is necessary. The documentation explains what this is for and it just seems cleaner without the tags. But feel free to push back.

Reply:
I'm alright with it as is. I was just thinking that in a package full of JSON-serializable structs, it would make it more obvious that this is not meant to be serialized and sent anywhere.

Reply:
I added them.